src/ffi/setobject.rs

```diff
@@ -65,42 +114,17 @@ pub unsafe fn PyAnySet_Check(ob: *mut PyObject) -> c_int {
         || PyType_IsSubtype(Py_TYPE(ob), &mut PyFrozenSet_Type) != 0) as c_int
 }
 
+// skipped PySet_CheckExact
+
+#[cfg_attr(windows, link(name = "pythonXY"))]
```

Comment:
I think no need for this attribute (the imported symbol is a function, not `static mut`)

```suggestion
```

Reply:
Same as above.